### ISSUE TITLE
"Open branch on remote"  in "Commit graph" requires an extra click in multi-folder workspace

### ISSUE THREAD

**Issue by MrLuje** [gitkraken/vscode-gitlens#2461]
### Description

1. Open a workspace with multiple folders in VSCode
2. Open "Commit graph" view
3. Right-click on a branch and select "Open branch on remote"
4. It asks on which repository to open the branch but I'm already in a context where it should have this information and shouldn't re-ask

### GitLens Version

13.2.0

### VS Code Version

Version: 1.74.3 (user setup)
Commit: 97dec172d3256f8ca4bfb2143f3f76b503ca0534
Date: 2023-01-09T16:59:02.252Z (1 wk ago)
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No

### Git Version

git version 2.34.1

### Logs, Screenshots, Screen Captures, etc

Here I am in a workspace with `monitoring` and `templates` folders
![image](https://user-images.githubusercontent.com/632075/213677489-60d86279-2e75-4877-96fd-94e67e0daa3b.png)

Trying to open `main` branch in `monitoring` repository
![image](https://user-images.githubusercontent.com/632075/213677124-107b93fc-c8f4-48af-a5f1-6f7a47fb8e57.png)

It asks on which repo but I'm already in `monitoring` repository
![image](https://user-images.githubusercontent.com/632075/213677192-44971ead-3b3b-43f3-af49-9047a4001f60.png)

**Comment by eamodio** [comment]
Is this still an issue? I can't reproduce it.

**Comment by MrLuje** [comment]
No, looks like it has been fixed at some point, I can't reproduce it anymore :)
Thanks
